db/structure.sql

```diff
@@ -67,6 +67,21 @@ CREATE TYPE public.resource_type AS ENUM (
 );
 
 
+--
+-- Name: reset_sequence(text, text, text); Type: FUNCTION; Schema: public; Owner: -
+--
+
+CREATE FUNCTION public.reset_sequence(tablename text, columnname text, sequence_name text) RETURNS void
+    LANGUAGE plpgsql
+    AS $$
+      DECLARE
+      BEGIN
+      EXECUTE 'SELECT setval( ''' || sequence_name  || ''', ' || '(SELECT MAX(' || columnname || ') FROM ' || tablename || ')' || '+1)';
+      END;
+
+    $$;
+
+
```

Comment:
Let's make sure not to commit changes to the database for PR's that don't need them - if something is off w/structure.sql, it might be worthwhile to create a separate PR